src/go/rpk/pkg/cli/cmd/cluster/storage/recovery/start.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package recovery
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newStartCommand(fs afero.Fs) *cobra.Command {
+	var (
+		topicNamePattern string
+		wait             bool
+		pollingInterval  time.Duration
+	)
+
+	cmd := &cobra.Command{
+		Use:   "start",
+		Short: "Start the recovery process",
+		Long: `Start the recovery process.
+		
+This command starts the process of restoring topics from the archival bucket.
+If the wait flag (--wait/-w) is set, the command will poll the status of the
+recovery process until it's finished.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			client, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			ctx := cmd.Context()
+
+			_, err = client.StartAutomatedRecovery(ctx, topicNamePattern)
```

Comment:
I don't know what all the potential return values will be. There's an example of a response [here](https://github.com/redpanda-data/redpanda/blob/83b69d048f514ab2cc06b2d8f9f34aab8c598b4c/src/v/redpanda/admin/api-doc/shadow_indexing.json#L56-L61), but it doesn't list examples for failure cases

I added the response to the code so it would be easy to modify if there's any special cases to handle later on. Though it there isn't, then it's just unneeded complexity to have it in the code and I think it's better to remove it. WDYT?

@abhijat Are there any responses from sending a `POST` to `/v1/cloud_storage/automated_recovery`, that we need to be specifically aware off?